pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -137,6 +147,42 @@ func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder
 	return resourceapply.ApplyConfigMap(client, recorder, configMap)
 }
 
+func checkForFeatureGates(featureGateLister configlistersv1.FeaturesLister) (map[string]bool, error) {
+	featureGateListConfig, err := featureGateLister.Get("cluster")
```

Comment:
do we know the cluster wide featureGate will be named cluster?

Reply:
Nope, I was using same name that we used for other CRs.